tests/Entity/TaxonomyTestTraitTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace weitzman\DrupalTestTraits\Tests\Entity;
+
+use Drupal\taxonomy\Entity\Vocabulary;
+use Drupal\Tests\RandomGeneratorTrait;
+use PHPUnit\Framework\TestCase;
+use weitzman\DrupalTestTraits\DrupalSetup;
+use weitzman\DrupalTestTraits\Entity\TaxonomyTestTrait;
+
+/**
+ * Tests the taxonomy creation trait.
+ */
+class TaxonomyTestTraitTest extends TestCase
```

Comment:
Now this should extend ExampleBase

Reply:
If these extend `ExampleBase`, it would re-run all tests defined in there for each of them. I was aiming for a more stripped down test for each trait.

Reply:
There aren't any tests defined in ExampleBase. Extending Examplebase does lead to running a bunch in setUp() and tearDown)_ but thats all fast enough IMO.